app/views/user_mailer/reservation_status_update.html.erb

```diff
@@ -0,0 +1,30 @@
+Hi <%= @reservation.reserver.name %>,
+<br />
+
+<%= render :partial => "#{@reservation.status.gsub(/\s/, '_')}" %>
+
+<br />
+
+<p><%= link_to 'Reservation Information', reservation_url(@reservation, only_path: false) %></p>
+
+<p>Equipment: <%= @reservation.equipment_model.name%> 
+<% if @reservation.equipment_object %> (<%= @reservation.equipment_object.name %>) <% end %></p>
+<p>Start Date: <%= @reservation.start_date.strftime(" %B %e, %Y at %l:%M%P")%> </p>
```

Comment:
there shouldn't be any time for the start and due dates, only for checkin and checkout